RecurrentFF/benchmarks/Moving-MNIST/moving_mnist.py

```diff
@@ -0,0 +1,268 @@
+import os
+import logging
+import time
+
+import torch
+from torch.utils.data import DataLoader, Dataset
+import wandb
+from matplotlib import pyplot as plt
+from multiprocessing import Process, Manager
+
+from RecurrentFF.model.model import RecurrentFFNet, TrainInputData, TrainLabelData, TestData
+from RecurrentFF.model.constants import EPOCHS, LEARNING_RATE, THRESHOLD, DAMPING_FACTOR, EPSILON, DEVICE
+
+NUM_CLASSES = 10
+INPUT_SIZE = 4096
+LAYERS = [1000, 1000, 1000]
+TRAIN_BATCH_SIZE = 5000
+TEST_BATCH_SIZE = 5000
+ITERATIONS = 10
+DATA_PER_FILE = 1000
+
+
+class MovingMNISTDataset(Dataset):
```

Comment:
The multiprocess dataloader part. Won't work for shuffling=True, but that is ok for me.